RLBotDiscordBot/cogs/admin.py

```diff
@@ -62,6 +62,46 @@ def __init__(self, bot):
                 json.dump(self.bot.settings, f, indent=4)
 
             await ctx.send(deleted_text)
+    
+    @commands.command()
+    async def add_whitelist(self, ctx, name, *, output):
+        if not self.check_perms(ctx):
+            return
+        
+        exists = name in self.bot.settings['whitelists']
+        if exists:
+            await ctx.send("Link already whitelisted")
+
+        self.bot.settings['whitelists'][name] = output
+        with open('./RLBotDiscordBot/settings.json', 'w') as file:
+            json.dump(self.bot.settings, file, indent=4)
+
+        settings = open('./RLBotDiscordBot/settings.json', 'r')
+        self.bot.settings = json.load(settings)
+        await ctx.send("Whitelist Added!")
+
+    @commands.command()
+    async def del_whitelist(self, ctx, *, name):
+        if not self.check_perms(ctx):
+            return
+
+        if name in self.bot.settings['whitelists']:
+
+            del self.bot.settings['whitelists'][name]
+            with open('./RLBotDiscordBot/settings.json', 'w') as f:
```

Comment:
Write a function for dumping bot settings to disk so you don't have to write this every time.